.github/workflows/greeting.yml

```diff
@@ -0,0 +1,15 @@
+name: Greetings
```

Comment:
At the moment this does not seem to add too much to the experience in my opinion. I understand that the checks are done behind the scenes at https://github.com/actions/first-interaction. I'm neutral about it. Just wonder if we could add more content if we need to add this. Maybe a link to Discord's mitiq channel.